client/src/Movies/Movie.js

```diff
@@ -2,10 +2,10 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
 const Movie = (props) => {
-  const [movie, setMovie] = useState({});
+  const [movie, setMovie] = useState();
  
   useEffect(() => {
-    const id = 1;
+    const id = props.match.params.id;
     // change ^^^ that line and grab the id from the URL
     // You will NEED to add a dependency array to this effect hook
 
```

Comment:
Those empty brackets are the dependency array. You will want to pass in the id to this array so that everytime the id changes. this useEffect hook runs and sets the movie to state.